en/Obsidian Publish/Set up a custom domain.md

```diff
@@ -1,48 +1,57 @@
 You can set up a custom domain or subdomain for your [[Introduction to Obsidian Publish|Obsidian Publish]] site.
 
-We don't yet have a way to provision SSL certificate on your behalf. If you want to enable HTTPS for a custom domain, you either need to use your own proxy or create a site on CloudFlare.
+At the moment, we only support configuring custom domains using the following methods:
+
+- [[#Set up using CloudFlare]] using [Full (strict) mode](https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/full-strict/).
+- [[#Set up using a proxy]]
+
+We don't yet have a way to provision SSL certificate on your behalf.
 
 ## Set up using CloudFlare
 
 The easiest way to set up a custom domain or subdomain is to create a [CloudFlare](https://cloudflare.com) account and let CloudFlare manage your domain's DNS.
 
-When you migrate to CloudFlare, you automatically keep all your existing DNS setup, including your subdomains, email service, and hosting services. Using CloudFlare can speed up your websites and add SSL to your sites for free.
+The following steps use CloudFlare to configure a custom domain for your Obsidian Publish site, either using a root domain (`mysite.com`) or an subdomain (`notes.mysite.com`).
 
-Typically, users host their Obsidian Publish content on a root domain (`mysite.com`) or an immediate subdomain (`notes.mysite.com`).
+> [!warning]
+> Obsidian Publish only supports custom domains through CloudFlare. Using the following the instructions with any other providers will likely not work.
```

Comment:
This reads like it's required to use Cloudflare.
Using Cloudflare is the only DNS based method we know of, but one could still use a proxy.

Reply:
```suggestion
> CloudFlare is the only officially supported provider for setting up custom domains. Using the following the instructions with any other providers will likely not work.
```

While there might be other providers that we don't know about yet, it doesn't change that CloudFlare is the only officially supported provider. Updated to try and reflect this.

Reply:
looks good.